### PR TITLE
PlotData2D example

### PR DESCRIPTION
**Pull request by sgkang**
Plotting un-gridded 2D data for scalar and vector fields.

### CODE REVIEW

**Comment by rowanc1** [comment]
Looks like you are plt showing?

image

**Comment by sgkang** [comment]
oh ... I see.


2016-08-23 8:51 GMT-06:00 Rowan Cockett <notifications@github.com>:

**Comment by sgkang** [comment]
Ready to be merged.

File: SimPEG/Utils/PlotUtils.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import numpy as np
+from scipy.interpolate import LinearNDInterpolator
+import matplotlib.pyplot as plt
+
+def plot2Ddata(xyz, data, vec=None, nx=100, ny=100,
```
Comment (lheagy) [inline SimPEG/Utils/PlotUtils.py]: vec=False ?

File: SimPEG/Utils/PlotUtils.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+               ax=None, mask=None, level=None, figname=None,
+               ncontour=10, dataloc=False, contourOpts={}, clim=None):
```
Comment (lheagy) [inline SimPEG/Utils/PlotUtils.py]: what is `dataloc`?

Comment (sgkang) [inline SimPEG/Utils/PlotUtils.py]: location of data

File: SimPEG/Utils/PlotUtils.py (hunk 1 above)
Comment (sgkang) [inline SimPEG/Utils/PlotUtils.py]: sounds good

**Comment by sgkang** [comment]
image

Any ideas?

**Comment by rowanc1** [comment]
You might need to rerun the example init function it looks like you have deleted a lot of the examples in the init.

**Comment by sgkang** [comment]
I guess I did run __init__.py, can we merge this branch to dev?

**Comment by sgkang** [comment]
It seems, I have not authorized to merge this pull request. Can some one do this?

File: SimPEG/Utils/PlotUtils.py [hunk 3]
```diff
@@ -0,0 +1,65 @@
+import numpy as np
+from scipy.interpolate import LinearNDInterpolator
+import matplotlib.pyplot as plt
+
+def plot2Ddata(xyz, data, vec=False, nx=100, ny=100,
+               ax=None, mask=None, level=None, figname=None,
+               ncontour=10, dataloc=False, contourOpts={}, clim=None):
+    """
+
+        Take unstructured xy points, interpolate, then plot in 2D
```
Comment (lheagy) [inline SimPEG/Utils/PlotUtils.py]: This docstring looks like it was copied from somewhere else?

Comment (lheagy) [inline SimPEG/Utils/PlotUtils.py]: is "mask" used at all in the code?

Comment (lheagy) [inline SimPEG/Utils/PlotUtils.py]: I have updated the docstring, but mask is still there unused

**Comment by lheagy** [comment]
I am good with this @rowanc1 : could you take a look?

**Comment by rowanc1** [comment]
👍

**Pull request merged**